packages/graphql-transformer-core/package.json

```diff
@@ -24,16 +24,17 @@
     "extract-api": "ts-node ../../scripts/extract-api.ts"
   },
   "dependencies": {
+    "@aws-amplify/graphql-transformer-interfaces": "2.1.0-beta.6",
```

Comment:
Where does this new dependency come from?

Reply:
Confirmed, this doesn't seem particularly harmful, it came to reduce duplication in types here [https://github.com/aws-amplify/amplify-category-api/commit/15bfb0bf11fcc24e29f27001412bbd1d5c88dd12#diff-fa5492731b531faeb[…]469d47359cd94d8e3cd0893](https://github.com/aws-amplify/amplify-category-api/commit/15bfb0bf11fcc24e29f27001412bbd1d5c88dd12#diff-fa5492731b531faeb6f346da9ea6e2a1d5481b2b7469d47359cd94d8e3cd0893)

Reply:
from this PR https://github.com/aws-amplify/amplify-category-api/pull/1044/files#diff-fa5492731b531faeb6f346da9ea6e2a1d5481b2b7469d47359cd94d8e3cd0893 .
I.e. this was needed to decouple cli-core from transformers-core impl.

Reply:
I.e. not use transformer core concrete types in cli core.